kombu/transport/redis_cluster.py

```diff
@@ -307,41 +316,75 @@ def _brpop_start(self, timeout=1):
             for _, _, conn, _ in self.connection.cycle._chan_to_sock:
                 if conn.key == key and conn.in_poll == False:
                     conn.in_poll = True
+                    conn.timeout = timeout
+                    if conn.key in self.ask_errors:
+                        del self.ask_errors[conn.key]
+                        conn.client.execute_command('ASKING')
```

Comment:
Don't we have to parse response of `asking` command?

Reply:
`execute_command` actually parses response